src/AuthService.ts

```diff
@@ -156,7 +156,10 @@ export class AuthService<TIDToken = JWTIDToken> {
         client_id: clientId,
         post_logout_redirect_uri: redirectUri
       }
-      const url = `${logoutEndpoint || `${provider}/logout`}?${toUrlEncoded(query)}`
+      const logoutUrl = `${logoutEndpoint || `${provider}/logout`}`;
+      // Check if the url already contains at least one parameter
+      const separator = new RegExp('^.*\?.+=.+$').test(logoutUrl) ?  '&' : '?';
```

Comment:
Hi, thank you for this code update. I actually just requested an enhancement for this without realising that this PR existed.

I think there we should make a very small tweak to the regex here. Instead of `'^.*\?.+=.+$'`, which will still use a '?' character for a query string parameter with no value, it should be changed to `'^.*\?.+=.*$'`. (Note the '*' instead of a '+' character before the '$' character.)

This will allow us to also add query string parameters with null/empty values (e.g. `?key=`)